scripts/mutation-test.sh

```diff
@@ -1,22 +1,36 @@
 #!/usr/bin/env bash
 
 set -eo pipefail
+oIFS="$IFS"; IFS=, ; set -- $1 ; IFS="$oIFS"
```

Comment:
what does this do?

Reply:
This line allows for parsing a comma separated list of a single argument (i.e takes `superfluid,twap` and makes it two separate inputs)